### PR TITLE
Add `--datatype` option to arangoimport

### PR DESCRIPTION
**Pull request by jsteemann**
### Scope & Purpose

Docs PR: https://github.com/arangodb/docs/pull/767

* Add `--datatype` startup option to arangoimport, in order to hard-code the
  datatype for certain attributes in the CSV/TSV import. For example, given
  the following input file:

      key,price,weight,fk
      123456,200,5,585852
      864924,120,10,9998242
      9949,70,11.5,499494
      6939926,2130,5,96962612

  When invoking arangoimport with the startup options

      --datatype key=string
      --datatype price=number
      --datatype weight=number
      --datatype fk=string

  it will turn the numeric-looking values in "key" into strings (so that they
  can be used in the `_key` attribute), but treat the attributes "price" and
  "weight" as numbers. The values in attribute "fk" finally will be treated as
  strings again (potentially because they are used for linking to other "_key"
  values).

- [ ] :hankey: Bugfix (requires CHANGELOG entry)
- [x] :pizza: New feature (requires CHANGELOG entry, feature documentation and release notes)
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification
- [x] :book: CHANGELOG entry made

#### Backports:

- [x] No backports required

#### Related Information

- [x] Docs PR: https://github.com/arangodb/docs/pull/767

### Testing & Verification

- [x] The behavior in this PR was *manually tested*
- [x] This PR adds tests that were used to verify all changes:
  - [x] Added new **integration tests** (e.g. in importing)

### CODE REVIEW

File: arangosh/Import/ImportFeature.cpp [hunk 1]
```diff
@@ -258,6 +265,24 @@ void ImportFeature::validateOptions(std::shared_ptr<options::ProgramOptions> opt
       FATAL_ERROR_EXIT();
     }
   }
+  for (auto const& it : _datatypes) {
+    auto parts = StringUtils::split(it, '=');
+    if (parts.size() != 2) {
+      TRI_ASSERT(parts.size() == 1);
```
Comment (cpjulia) [inline arangosh/Import/ImportFeature.cpp:271]: should this be addressed as well in lines 465 and 466, where it's verified again if there's an assignment expression within the string?

**Review by cpjulia — APPROVED** [review]
LGTM

File: arangosh/Import/ImportFeature.cpp (hunk 1 above)
Comment (cpjulia) [reply arangosh/Import/ImportFeature.cpp:271]: I mean, if there's no assignment, to assert `parts.size() == 1`

**Review by cpjulia — APPROVED** [review]
LGTM

File: arangosh/Import/ImportFeature.cpp [hunk 2]
```diff
@@ -142,10 +142,16 @@ void ImportFeature::collectOptions(std::shared_ptr<options::ProgramOptions> opti
                      "translate an attribute name (use as --translate "
                      "\"from=to\", for csv and tsv only)",
                      new VectorParameter<StringParameter>(&_translations));
+  
+  options->addOption("--datatype",
+                     "force a specific datatype for an attribute (use as "
+                     "\"attribute=type\", for csv and tsv only, takes precendence over --convert)",
```
Comment (Simran-B) [inline arangosh/Import/ImportFeature.cpp:148]:
```suggestion
                     "force a specific datatype for an attribute "
                     "(null/boolean/number/string). Use as \"attribute=type\". "
                     "For CSV and TSV only. Takes precendence over --convert",
```

File: arangosh/Import/ImportFeature.cpp [hunk 3]
```diff
@@ -142,10 +142,16 @@ void ImportFeature::collectOptions(std::shared_ptr<options::ProgramOptions> opti
...
+                     new VectorParameter<StringParameter>(&_datatypes))
+                     .setIntroducedIn(30900);
 
   options->addOption("--remove-attribute",
-                     "remove an attribute before inserting an attribute"
-                     " into a collection (for csv and tsv only)",
+                     "remove an attribute before inserting documents"
+                     " into collection (for csv and tsv only)",
```
Comment (Simran-B) [inline arangosh/Import/ImportFeature.cpp:154]:
```suggestion
                     " into collection (for CSV and TSV only)",
```

File: CHANGELOG [hunk 4]
```diff
@@ -1,6 +1,29 @@
 devel
 -----
 
+* Add `--datatype` startup option to arangoimport, in order to hard-code the
+  datatype for certain attributes in the CSV/TSV import. For example, given
+  the following input file:
```
Comment (Simran-B) [inline CHANGELOG:6]:
```suggestion
  datatype (null/boolean/number/stirng) for certain attributes in the CSV/TSV import.
  For example, given the following input file:
```

**Review by Simran-B — APPROVED** [review]

**Pull request merged**